### DISCUSSION TITLE
Ribbon icon keeps appearing after every update

### DISCUSSION THREAD

**Discussion by gathering-moss** [zsviczian/obsidian-excalidraw-plugin#1655] [Q&A]
I like to keep my ribbon menu icons to a minimum. I could have sworn that there used to be a setting for Excalidraw to keep its icon out of the ribbon menu, and now I don't see that option any more. Now, each time I update Excalidraw, its ribbon menu icon reappears, and I have to remove it from the ribbon menu in Appearance settings.

Did there used to be a setting to not use the ribbon menu icon? If I am completely mistaken, is there any way I can prevent the Excalidraw ribbon menu icon from appearing after every update?

**Comment by Calamitous** [comment]
Having the same issue here.  I have my most-commonly-used button at the bottom of the ribbon, and after every update I end up accidentally creating some empty excalidraw docs because unexpectedly there's a new button there.  

I'm not sure why this even happens.  If I wanted the button on the ribbon, I'd right-click and put it on there.

**Reply by how2051** [reply]
Me too! My icon used to be at a fixed position, but every time I reopen Obsidian, this Excalidraw icon refresh and go to the bottom.
:(

**Comment by semioniy** [comment]
Same issue. Excalidraw icon reappears in the ribbon all the time. It's not even after an update - after I restart Obsidian, excalidraw icon reappears.

**Reply by how2051** [reply]
I checked just now, I don't have this issue now, maybe u just need to update Excalidraw to the newest version, mine is 2.6.8, and this version fix some icon problem

**Reply by semioniy** [reply]
True, thanx for the heads-up!
